packages/fortress/views/download.pug

```diff
@@ -0,0 +1,12 @@
+extends layout
+
+block content
+  h1 #{data.global.title} <span class="highlight">#{data.products[index].title}</span>
```

Comment:
If you want to be a pug extraordinaire, you can also [set a span this way](https://pugjs.org/language/interpolation.html#tag-interpolation).